src/convertToObject.js

```diff
@@ -1,18 +1,27 @@
 'use strict';
 
 /**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssObj = {};
+
+  sourceString
+    .split(';')
+    .forEach((element) => {
+      const [property, value] = element.split(':');
+
+      if (property && value) {
+        const normalizedProperty = property.replace('\n', '').trim();
+
+        cssObj[normalizedProperty] = value.trim();
+      }
+    });
```

Comment:
You can use one method that returns an object immediately. 
So you don`t need a declaration above ( 10 line )

Reply:
Object.assign?